shared/common-adapters/list-item2.tsx

```diff
@@ -58,22 +59,15 @@ const HoverBox = Styles.isMobile
       },
     })
 
-const containerStyle = (props: Props) =>
-  props.type === 'Small'
-    ? props.statusIcon
-      ? styles.contentContainerSmallWithStatusIcon
-      : styles.contentContainerSmall
-    : props.statusIcon
-    ? styles.contentContainerLargeWithStatusIcon
-    : styles.contentContainerLarge
-const iconStyle = (props: Props) =>
-  props.type === 'Small'
-    ? props.statusIcon
-      ? styles.iconSmallWithStatusIcon
-      : styles.iconSmall
-    : props.statusIcon
-    ? styles.iconLargeWithStatusIcon
-    : styles.iconLarge
+// Using margin and keeping the icon on the let using absolute is to work around an issue in RN where
+// the nested views won't grow and still retain their widths correctly
+const iconLeftStyle = (props: Props) => (props.statusIcon ? {left: statusIconWidth} : null)
```

Comment:
the previous implementation existed so the styles would be static since these item are reused a lot and there are thousands of them being rendered. let's not make this change which makes this style change on every render